cnf-tests/testbinaries/oslat-runner/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+	"os/exec"
+	"strings"
+	"time"
+
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+)
+
+func main() {
+	klog.InitFlags(nil)
+
+	var oslatStartDelay = flag.Int("oslat-start-delay", 0, "Delay in second before running the oslat binary, can be useful to be sure that the CPU manager excluded the pinned CPUs from the default CPU pool")
+	var rtPriority = flag.String("rt-priority", "1", "Specify the SCHED_FIFO priority (1-99)")
+	var runtime = flag.String("runtime", "10m", "Specify test duration, e.g., 60, 20m, 2H")
+
+	flag.Parse()
+
+	selfCPUs, err := getSelfCPUs()
+	if err != nil {
+		klog.Fatalf("failed to get self allowed CPUs: %v", err)
+	}
+
+	if selfCPUs.Size() < 2 {
+		klog.Fatalf("the amount of requested CPUs less than 2, the oslat requires at least 2 CPUs to run")
+	}
+
+	mainThreadCPUSet := cpuset.NewCPUSet(selfCPUs.ToSlice()[0])
+	updatedSelfCPUs := selfCPUs.Difference(mainThreadCPUSet)
+
+	printNodeInformation()
+
+	if *oslatStartDelay > 0 {
+		time.Sleep(time.Duration(*oslatStartDelay) * time.Second)
+	}
+
+	oslatArgs := []string{
+		"--runtime", *runtime,
+		"--rtprio", *rtPriority,
+		"--cpu-list", updatedSelfCPUs.String(),
+		"--cpu-main-thread", mainThreadCPUSet.String(),
+	}
+
+	klog.Infof("Running the oslat command with arguments %v", oslatArgs)
+	out, err := exec.Command("/usr/bin/oslat", oslatArgs...).CombinedOutput()
```

Comment:
I think we should have a parameter to set the full path of `oslat`. Is fine to use this value as default.

Reply:
will add const